IMDB.py

```diff
@@ -0,0 +1,153 @@
+"""
+В файле хранятся данные с сайта IMDB. Скопированные данные хранятся в файле
+./data5/ ratings.list.
+a.	Откройте и прочитайте файл(если его нет необходимо вывести ошибку).
+b.	Найдите ТОП250 фильмов и извлеките заголовки.
+c.	Программа создает 3 файла  top250_movies.txt
+– названия файлов, ratings.txt – гистограмма рейтингов,
+years.txt – гистограмма годов.
+"""
+
+import os
+
+END_OF_OUR_ZONE_OF_INTEREST = 278
+TRASH_IN_THE_BEGINNING = 26
+
+try:
+    # open the original file
+    f = open('ratings.list', 'r')
+    # variable for working with text
+    a = ''
+    # counter to perform the desired number of repetitions
+    counter1 = 1
+    for i in f:
+        while counter1 < END_OF_OUR_ZONE_OF_INTEREST:
+            a = str(f.readline())
+            top250 = open('top250.txt', 'a')
+            top250.write(a)
+            top250.close()
+            counter1 += 1
+    f.close()
+
+    top250 = open('top250.txt', 'r')
+    counter2 = 1
+    # skip the lines at the beginning of the file
+    for i in top250:
+        while counter2 < TRASH_IN_THE_BEGINNING:
+            a = str(top250.readline())
+            counter2 += 1
+        a = str(top250.readline())
+        while a != '':
+            top250_temp = open('top250_movies_temp.txt', 'a')
+            a = str(top250.readline())
+            top250_temp.write(a)
```

Comment:
why can't we keep it in memory

Reply:
when developing a program, it was very convenient to observe the output of the program in a file, for me it is more convenient

Reply:
but as I understand you've finished debug. it's a good idea to delete debug code before sending result.